src/lib/server/websearch/scrape/playwright.ts

```diff
@@ -64,7 +64,7 @@ export async function withPage<T>(
 
 	try {
 		const page = await ctx.newPage();
-		await blocker.enableBlockingInPage(page);
+		process.env.PLAYWRIGHT_ADBLOCKER && (await blocker.enableBlockingInPage(page));
```

Comment:
```suggestion
		(/true/i).test(process.env.PLAYWRIGHT_ADBLOCKER) && (await blocker.enableBlockingInPage(page));
```

Since if you set `PLAYWRIGHT_ADBLOCKER=false` it will actually be `"false"` which evaluates to true since it's not an empty string

Other than than LGTM!